src/webClient/index.js

```diff
@@ -98,7 +107,10 @@ const postQuery = (query) => {
       throw new Error('Error with posting query');
     })
     .then(res => res.json())
-    .then(obj => obj.id)
+    .then(obj => {
+      updateSession(obj.session_id);
```

Comment:
Updating the state's session value using the session_id of the initial post's json response